### PR TITLE
php@7.3 7.3.23

### PR DESCRIPTION
**Pull request by derrabus**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Reported at https://bugs.php.net/bug.php?id=80171

**Comment by derrabus** [comment]
Interesting that the php 7.4.11 build succeeded though.

**Comment by SMillerDev** [comment]
The bug report explains why, this feature was dropped in php 7.4

**Comment by derrabus** [comment]
The formula builds fine on my MBP. Is there anything I can do to help with the issue?

Bildschirmfoto 2020-10-02 um 15 07 45

**Comment by shivammathur** [comment]
@derrabus The build is failing because `Xcode 12` is now the default version `macos-10.15` in GitHub Actions.
Clang with `Xcode 12` has `-Werror=implicit-function-declaration` flag on by default. 
https://developer.apple.com/documentation/xcode-release-notes/xcode-12-release-notes#Resolved

I was able to compile it by switching to `Xcode 11.7` in my tap [here](https://github.com/shivammathur/homebrew-php/runs/1196898659?check_suite_focus=true).

**Comment by derrabus** [comment]
I see, so the failure is unrelated to the version bump, it's the CI environment that has changed. Thank you for the clarification. 😃

**Comment by derrabus** [comment]
I have added @nikic's commit as patch to the formula so we can see if the CI is now able to compile.

File: Formula/php@7.3.rb [hunk 1]
```diff
@@ -50,6 +50,11 @@ class PhpAT73 < Formula
   uses_from_macos "libxslt"
   uses_from_macos "zlib"
 
+  patch do
+    url "https://github.com/php/php-src/commit/aa405b7da270595d349d0596ad31305a41d4b1c0.patch"
```
Comment (SMillerDev) [inline Formula/php@7.3.rb:54]:
```suggestion
    url "https://github.com/php/php-src/commit/aa405b7da270595d349d0596ad31305a41d4b1c0.patch?full_index=1"
```

Comment (derrabus) [reply Formula/php@7.3.rb]: Done. One of the github actions has already yelled at me about that. 😉

File: Formula/php@7.3.rb [hunk 2]
```diff
@@ -50,6 +50,13 @@ class PhpAT73 < Formula
   uses_from_macos "libxslt"
   uses_from_macos "zlib"
 
+  # Fix build issues with Xcode 12.
+  # see https://bugs.php.net/bug.php?id=80171
+  patch do
+    url "https://github.com/php/php-src/commit/aa405b7da270595d349d0596ad31305a41d4b1c0.patch?full_index=1"
+    sha256 "6468bdc5fc7bc8f623af1ec2df34d0e0f7a689cc5f2c26552e0d97225c93b0b0"
```
Comment (SMillerDev) [inline Formula/php@7.3.rb:57]:
```suggestion
    sha256 "4c51e35fd936a7f3f5613c72e6395b9afa8f569061c00849683e09e8fe986a0f"
```

**Comment by derrabus** [comment]
The build is green now. 🎉

**Comment by SMillerDev** [comment]
Can you squash the commits into one `php@7.3 7.3.23`

**Comment by derrabus** [comment]
Sure, done.

**Review by SMillerDev — APPROVED** [review]
Thanks @derrabus  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by derrabus** [comment]
Thanks for the help, @SMillerDev, and @nikic for the quick patch. 😃

**Pull request closed without merging**